client/src/main/java/io/split/engine/sse/dtos/FeatureFlagChangeNotification.java

```diff
@@ -1,13 +1,20 @@
 package io.split.engine.sse.dtos;
 
 import io.split.engine.sse.NotificationProcessor;
+import io.split.engine.sse.enums.CompressType;
 
-public class SplitChangeNotification extends IncomingNotification {
+public class FeatureFlagChangeNotification extends IncomingNotification {
     private final long changeNumber;
+    private final long previousChangeNumber;
+    private final String data;
+    private CompressType compressType;
```

Comment:
I think this one can be `final` as well.
```suggestion
    private final CompressType compressType;
```

Reply:
CompressType can be null.